docs/src/architecture/mapper.md

```diff
@@ -1,9 +1,97 @@
-# The mapper
+# The tedge-mapper
 
-Or more appropriately the mappers,
-because there is a specific mapper for each cloud provider,
-a Cumulocity mapper, an Azure mapper.
-
-The purpose of these mappers is to translate
-messages written using the cloud-agnostic [Thin-Edge Json format](thin-edge-json.md),
+The tedge-mapper is a key concept to support multiple cloud providers.
+The purpose is to translate
+messages written using the cloud-agnostic [Thin Edge JSON format](thin-edge-json.md),
 into cloud-specific messages.
+
+The tedge-mapper is composed of multiple cloud-specific mappers, such as Cumulocity mapper and Azure mapper.
+Each mapper is responsible for its dedicated cloud.
+These specific mappers are launched by the respective `tedge connect` command.
+For instance, `tedge connect c8y` establishes a bridge to Cumulocity and launches a Cumulocity mapper
+that translates the messages in the background.
+
+A mapper subscribes to the reserved MQTT topic `tedge/measurements` with the QoS level 1 (at least once).
+The messages that arrive in the mapper should be formed in the [Thin Edge JSON](thin-edge-json.md) format.
```

Comment:
We should change the use of Thin Edge JSON consistently to thin-edge.io JSON. But Ok for now. We need to change that together in all docs then.